tests/test_pythonpackage_basic.py

```diff
@@ -87,9 +89,25 @@ def test_get_dep_names_of_package():
     dep_names = get_dep_names_of_package("python-for-android")
     assert "colorama" in dep_names
     assert "setuptools" not in dep_names
-    dep_names = get_dep_names_of_package("python-for-android",
-                                         include_build_requirements=True)
-    assert "setuptools" in dep_names
+    try:
+        dep_names = get_dep_names_of_package(
+            "python-for-android", include_build_requirements=True,
+            verbose=True,
+        )
+    except NotImplementedError as e:
```

Comment:
I'd be tempted to make an `IsWheelError` subclass or something, on the basis that specific exceptions aren't used often enough, but I'm never really sure about this.

Reply:
@inclement I'm somehow afraid somebody will use that to test if a package is a wheel and then end up with broken code once it is actually implemented and this no longer returns an error (or they use it right but we remove the error so it can no longer be imported once this is implemented and therefore no longer ever thrown). So I think I'd rather stick with a `NotImplementedError` here, personally

Reply:
@inclement what are your thoughts on my thought on this? :stuck_out_tongue: other than this I think it's ready for merge

Reply:
I'm merging, we can always eventually address that one later if we feel it's critical

Reply:
I wouldn't have approved the PR if I thought this needed changing, sorry if that wasn't clear and thanks for explaining the motivation!